common/src/main/java/com/wynntils/features/user/map/MinimapFeature.java

```diff
@@ -234,9 +248,19 @@ private void renderPois(
             }
 
             float currentZoom = 1f / scale;
-            for (Poi poi : Stream.concat(MapModel.getServicePois().stream(), MapFeature.INSTANCE.customPois.stream())
-                    .toList()) {
 
+            List<Poi> poisToRender = new ArrayList<>(MapModel.getServicePois());
+            poisToRender.addAll(MapFeature.INSTANCE.customPois);
+            List<PlayerMiniMapPoi> playerPois = HadesUserModel.getHadesUserMap().values().stream()
+                    .filter(user -> user.isPartyMember() && renderRemotePartyPlayers
+                            || user.isMutualFriend() && renderRemoteFriendPlayers)
+                    .sorted(Comparator.comparing(
+                            hadesUser -> hadesUser.getMapLocation().getY()))
```

Comment:
Do we need to sort this? Why are minimap pois not sorted? @Incompleteusern

Reply:
This was just something I copied from the main map feature, and it made sense to leave that sort.

It appears that the sort here matters but I can test that. Unless I missed the logic somewhere I don't see any sort logic elsewhere for these.

Reply:
Oh. We don't even sort any other pois on the main map now. This is fine as is then.